### ISSUE TITLE
Request: [once] abstraction in vanilla

### ISSUE THREAD

**Issue by megrimm** [pure-data/pure-data#1097]
Any chance we could get [once] in vanilla? I use it all the time working with intro classes and it is kind of a messy one for them to build when really just triggering a [readsf~] or whatever with a button.

**Comment by porres** [comment]
where is this external?

**Comment by megrimm** [comment]
idk... @umlaeute has an abstract floating around in a number of libraries I think at this point. iemlib, purepd, etc. 
@porres i mean you have [onebang] in cyclone right? i would say a simple abstraction in extra folder would be fine.

**Comment by porres** [comment]
In the help file of [bng] there's an example of something like this. I just read your other email, if this is for workshops and stuff, just tell students to look up this example and copy/paste, might also be a nice example on show them how to make an abstraction, or, yeah, teach them how to install externals is also a nice first step. I'm sorry cause imho I just think this is just too simple to bother.

**Comment by porres** [comment]
> In the help file of [bng] there's an example of something like this.

<img width="887" alt="Screen Shot 2020-07-27 at 13 23 06" src="https://user-images.githubusercontent.com/5360142/88566401-53108d00-d00c-11ea-86f4-6978ca2cf070.png">

**Comment by porres** [comment]
> @porres i mean you have [onebang] in cyclone right? 

we all have it :) and max has it, so cyclone has it

what "I" have is else/nbang ;) which seems like a better and more interesting/useful design for me

**Comment by megrimm** [comment]
yeah i know. i was torn putting the request in because of the simplicity. just thought i would give it a try anyway. I am always trying to minimize the complexities as much as possible because they take away from class time and give me a headache :)

so the request is more just a thought i guess being in the right in the middle of the headache.

i think the [keyup] issue listed here is more pressing https://github.com/pure-data/pure-data/issues/945

**Comment by porres** [comment]
> i think the [keyup] issue listed here is more pressing #945

tell me about it... I'll try my best to see if I can work on a solution myself ;)

**Comment by porres** [comment]
can we close this @megrimm ?

**Comment by umlaeute** [comment]
please fix the example first (as mentioned in #1436 )

**Comment by porres** [comment]
> please fix the example first (as mentioned in #1436 )

done

**Comment by porres** [comment]
can we close this?
